spec/requests/apartments_spec.rb

```diff
@@ -263,11 +324,25 @@
         bedrooms: 3,
         bathrooms: 4,
         pets: "spiders only",
-        user_id: user.id
-      }}
-
-      post '/apartments', params: apartment_params
+        image: "https://images.unsplash.com/photo-1560448204-e02f11c3d0e2?q=80&w=1470&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D")
+        
+      apartment_params = {
+        apartment: {
```

Comment:
This test could be expanded upon to ensure each individual attribute is checked.  Making all updated attributes empty is a very vague test and won't tell you which attribute is truly throwing a 422 error.